build.gradle

```diff
@@ -374,17 +378,43 @@ pmd {
     ruleSets = []
 }
 
-findbugs {
-    toolVersion = findbugsVersion
-    visitors = [
-        "FindDeadLocalStores"
-    ]
+spotbugs {
+    reportLevel = "low"
+    toolVersion = spotbugsVersion
+    includeFilter = file("$rootProject.projectDir/static-analysis/teammates-spotbugs.xml")
 }
 
-tasks.withType(FindBugs) {
+tasks.withType(com.github.spotbugs.SpotBugsTask) {
     reports {
         xml.enabled = false
-        html.enabled = true
+        html.enabled = false
+        emacs.enabled = true
+    }
+}
+
+spotbugsMain {
+    finalizedBy "printSpotbugsMainViolations"
+}
+
+task printSpotbugsMainViolations {
+    onlyIf {
+        file("build/reports/spotbugs/main.emacs").exists()
```

Comment:
Use `${buildDir}` instead of `build` as we change `buildDir` somewhere in the gradle script?

Reply:
@xpdavid done.